### ISSUE TITLE
Forecast data is offset +1 day after Midnight GMT

### ISSUE THREAD

**Issue by notjj** [Makin-Things/platinum-weather-card#56]
**Checklist:**

- [X] I updated to the latest version available
- [X] I cleared the cache of my browser

**Release with the issue: 0.0.4, 0.0.5b7**

**Last working release (if known):**

**Browser and Operating System: Chrome, OS X 12.3.1**

**Description of problem:**

When past GMT midnight, forecast entries become offset by +1 day. (Monday's forecast shows Tuesday's data). 

Datasource is OpenWeatherMap in daily forecast mode.

Screenshots taken at 22:02 GMT-7:00, Oct 9th

**Card Configuration:**
```
 card_config_version: 8
entity_apparent_temp: sensor.apparent_temperature
entity_forecast_icon: sensor.openweathermap_condition
entity_forecast_icon_1: weather.openweathermap
entity_forecast_max: sensor.openweathermap_forecast_temperature
entity_forecast_max_1: weather.openweathermap
entity_forecast_min: sensor.openweathermap_forecast_temperature_low
entity_forecast_min_1: weather.openweathermap
entity_humidity: sensor.relative_humidity
entity_pop: sensor.openweathermap_forecast_precipitation_probability
entity_pop_1: weather.openweathermap
entity_pos: sensor.openweathermap_forecast_precipitation
entity_pos_1: weather.openweathermap
entity_pressure: sensor.pressure
entity_summary_1: weather.openweathermap
entity_sun: sun.sun
entity_temperature: sensor.temperature
entity_wind_bearing: sensor.wind_direction
entity_wind_gust: sensor.wind_gust
entity_wind_speed: sensor.wind_speed
option_static_icons: true
option_time_format: 24hour
section_order:
  - overview
  - extended
  - slots
  - daily_forecast
show_section_extended: false
slot_l1: forecast_max
slot_l2: forecast_min
slot_l3: wind
slot_l4: sun_next
slot_l5: remove
slot_l6: remove
slot_l7: remove
slot_l8: remove
slot_r1: popforecast
slot_r2: humidity
slot_r3: pressure
slot_r4: sun_following
slot_r5: remove
slot_r6: remove
slot_r7: remove
slot_r8: remove
type: custom:platinum-weather-card

```

**Additional information:**
![Screen Shot 2022-10-09 at 9 55 03 PM](https://user-images.githubusercontent.com/7884822/194801848-d89f952a-efe1-4dd4-9bd5-09a3717cd70d.png)
![Screen Shot 2022-10-09 at 9 55 39 PM](https://user-images.githubusercontent.com/7884822/194801856-d617c86d-48cf-4d3b-858e-001f7da747da.png)

**Comment by Makin-Things** [comment]
Please test 0.0.5b8. Hopefully it fixes it, but it is really difficult for me to test thoroughly.

**Comment by Makin-Things** [comment]
Believed to be fixed in 1.0.0. Please reopen if the problem still exists.

**Comment by notjj** [comment]
Sorry for the long wait… 0.0.5b8 did resolve the issue, the weather forecast is now consistent even past midnight gmt.
